src/ZendDeveloperTools/Profiler.php

```diff
@@ -209,7 +209,7 @@ public function collect(MvcEvent $mvcEvent)
         if (isset($this->collectors)) {
             foreach ($this->collectors as $collector) {
                 $collector->collect($mvcEvent);
-                $this->report->addCollector(unserialize(serialize($collector)));
+                $this->report->addCollector($collector);
```

Comment:
I think the change is acceptable, since we're not storing collectors (which is a shame :-( )